app/src/server/api/routers/datasetEntries.router.ts

```diff
@@ -466,6 +498,46 @@ export const datasetEntriesRouter = createTRPCRouter({
       return success("Dataset entries deleted");
     }),
 
+  relabel: protectedProcedure
+    .input(
+      z.object({
+        ids: z.string().array(),
+      }),
+    )
+    .mutation(async ({ input, ctx }) => {
+      if (input.ids.length === 0) {
+        return error("No ids provided");
+      }
+      const { dataset } = await prisma.datasetEntry.findUniqueOrThrow({
+        where: { id: input.ids[0] },
+        include: {
+          dataset: true,
+        },
+      });
+
+      if (!dataset) {
+        return error("Dataset not found for dataset entry");
+      }
+
+      await requireCanModifyProject(dataset.projectId, ctx);
+
+      const batchId = Date.now().toString();
```

Comment:
Can we use `new Date().toISOString()` here instead? Will be more interpretable when it shows up in the filters.

Reply:
Works for me